nextcord/application_command.py

```diff
@@ -519,9 +672,7 @@ def _from_callback(self, callback: Callable) -> ApplicationSubcommand:
     @property
     def payload(self) -> dict:
         """Verifies the content of the ApplicationSubommand then constructs and returns the payload for this subcommand.
-
         This does not return a complete application command payload for Discord, only the subcommand portion of it.
-
         Returns
```

Comment:
Why have these blank lines been removed? Removing them only serves to make this less readable.